new/LIT6237GadlaLanginos.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6237GadlaLanginos" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ገድል፡ ዘአቡ፡ ለንጊኖስ፡ ንቡረ፡ እድ፡ ዘደብረ፡ ማህው፡</title>
+                <title type="normalized" xml:lang="gez" corresp="#t2">Gadl za-ʾabu Langinos nǝbura ʾǝd za-Dabra Māhǝw</title>
```

Comment:
I wonder if it would make sence to mark up the word ዘአቡ፡ with `<sic>`

Reply:
Well, so far it hasn't been done and the phenomenon is not unheard of check e. g. LIT4835Martyr

Reply:
I see :) You are right!